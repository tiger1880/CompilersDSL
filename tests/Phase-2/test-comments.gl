$$ this is also a comment 
$$ 
    $$ hello $$
    bye
$$

$ line comment