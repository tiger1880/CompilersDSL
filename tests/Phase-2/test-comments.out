BLOCK COMMENT: $$ this is also a comment 
$$
LINE COMMENT: $$ hello $$
IDENTIFIER: bye
LINE COMMENT: $$
LINE COMMENT: $ line comment
